tests/lineage/test_lineage.py

```diff
@@ -85,26 +85,26 @@ def test_lineage(self, _get_backend):
         # post process with no backend
         post = apply_lineage(func)
         post(op1, ctx1)
-        self.assertEqual(send_mock.call_count, 1)
-        send_mock.reset_mock()
+        # self.assertEqual(send_mock.call_count, 1)
+        # send_mock.reset_mock()
```

Comment:
I have disabled sending to a backend at the moment. This will be fixed in a follow up. I first want to get it right IN Airflow then work on external connections (Apache Atlas, Amundsen etc)